test/e2e/kubectl.go

```diff
@@ -112,6 +112,14 @@ var (
 )
 
 var _ = KubeDescribe("Kubectl client", func() {
+	// localized helper function, not meant to be reused outside of this test.
+	forEachPod := func(framework *Framework, selectorKey string, selectorValue string, fn func(api.Pod)) {
```

Comment:
Rather than making ambiguous wrappers hidden in random tests, if a test needs syntactic sugar for a reused ClusterVerification, they can just declare it locally in 5 lines of code at the top of their test.